src/storage.js

```diff
@@ -275,9 +276,53 @@ class AWSStorage {
       }
     });
 
-    const result = await this.client.send(new PutObjectCommand(input));
-    log.info(`Object uploaded to: ${this.bucket}/${key}`);
-    return result;
+    try {
+      await this.client.send(new PutObjectCommand(input));
+      log.info(`Object uploaded to: ${this.bucket}/${key}`);
+    } catch (e) {
+      log.error(`Unable to upload object to: ${this.bucket}/${key}`, e);
+      return new Response('', {
+        status: e.$metadata.httpStatusCode,
+      });
```

Comment:
Adding a response that shows an error without revealing too much detail

Reply:
I think this would be ok.... I don't think the bucketid will be a great secret.